libs/common/src/utils/ipfs.client.ts

```diff
@@ -6,7 +6,7 @@ import FormData from 'form-data';
 import { extension as getExtension } from 'mime-types';
 import { CID } from 'multiformats/cid';
 import { blake2b256 as hasher } from '@multiformats/blake2/blake2b';
-import { base58btc } from 'multiformats/bases/base58';
+import { base32 } from 'multiformats/bases/base32';
```

Comment:
are we doing base58btc?

Reply:
so we are doing base32 for IPFS CID but base58btc for dsnp hash